src/client_side_request.cc

```diff
@@ -1111,7 +1112,7 @@ clientInterpretRequestHeaders(ClientHttpRequest * http)
         }
 
 #if USE_FORW_VIA_DB
-        fvdbCountVia(s.termedBuf());
+        fvdbCountVia(StringToSBuf(s));
```

Comment:
This change does not belong to this PR, right? Some kind of merge problem? Please check for others.

Reply:
Yes, it's something spurious that sneaked in when rebasing, I think. I'll recheck the whole PR

Reply:
I needed to rebase